xml/System.IO/WindowsRuntimeStreamExtensions.xml

```diff
@@ -48,10 +80,31 @@
         <Parameter Name="stream" Type="System.IO.Stream" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="stream">The stream to convert.</param>
+        <summary>Converts a managed stream in the .NET for Windows 8.x Store apps to an input stream in the Windows Runtime.</summary>
+        <returns>A Windows Runtime <see xref="T:Windows.Storage.Streams.IInputStream"/> object that represents the converted stream.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+          ## Remarks
+          
+> [!NOTE]
+> In Visual Basic and C#, you can call this method as an instance method on any object of type [Stream](/dotnet/api/system.io.stream). When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](/dotnet/visual-basic/programming-guide/language-features/procedures/extension-methods) or [Extension Methods (C# Programming Guide)](/dotnet/csharp/programming-guide/classes-and-structs/extension-methods).
```

Comment:
Please use the ~/docs format for conceptual content that is under the dotnet path so links get validated at build time. Apply similar changes to the rest of the PR.

```suggestion
> In Visual Basic and C#, you can call this method as an instance method on any object of type <System.IO.Stream>. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](~/docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md) or [Extension Methods (C# Programming Guide)](~/docs/csharp/programming-guide/classes-and-structs/extension-methods.md).
```